lldb/include/lldb/Interpreter/CommandInterpreter.h

```diff
@@ -766,6 +768,12 @@ class CommandInterpreter : public Broadcaster,
   CommandUsageMap m_command_usages;
 
   StreamString m_transcript_stream;
+
+  /// Contains a list of handled commands, output and error. Each element in
+  /// the list is a dictionary with three keys: "command" (string), "output"
+  /// (list of strings) and optionally "error" (list of strings). Each string
+  /// in "output" and "error" is a line (without EOL characters).
```

Comment:
If we have a transcript already, can we start storing the time it took the run each command and also add this to the JSON as a "time_sec" (to indicate time in seconds) as a float value?

Reply:
Yes, we definitely can. I plan to do that as a separate PR as a follow-up.

I want to land this PR sooner rather than later, in order to unblock my other work in Meta (to log this transcript into the `lldb_usage` table).